src/bootstrap/src/ferrocene/run/update_certified_core_symbols.rs

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+// SPDX-FileCopyrightText: The Ferrocene Developers
+
+use std::fs;
+
+use build_helper::symbol_report::SymbolReport;
+
+use super::CertifiedCoreSymbols;
+use crate::builder::{Builder, RunConfig, ShouldRun, Step};
+use crate::core::config::TargetSelection;
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub(crate) struct UpdateCertifiedCoreSymbols {
+    host: TargetSelection,
+}
+
+pub(crate) const TRACKED_FILE: &str = "ferrocene/doc/symbol-report.json";
+
+impl Step for UpdateCertifiedCoreSymbols {
+    type Output = ();
+    const DEFAULT: bool = false;
+    const IS_HOST: bool = true;
+
+    fn should_run(run: ShouldRun<'_>) -> ShouldRun<'_> {
+        run.alias("update-certified-core-symbols")
+    }
+
+    fn make_run(run: RunConfig<'_>) {
+        run.builder.ensure(UpdateCertifiedCoreSymbols { host: run.build_triple() });
```

Comment:
This doesn't look right. This always runs on the host, which means it can have different output between Linux and macOS. we should consistently run for a single target (Linux seems fine since it's easy to cross-compile to).

I would remove `host` altogether and hard-code linux in the `run` step.